{
    "docstring": "\"\"\"Perform the same as __call__(), without catching the exception.\"\"\"",
    "name": "debug",
    "namespace": "django.test.testcases.SimpleTestCase.debug",
    "class_name": "",
    "module_name": "",
    "package_name": ""
}